pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -241,6 +241,7 @@ func TestValidatePolicy(t *testing.T) {
 			name:     "no weight defined in policy",
 			policy:   config.Policy{Priorities: []config.PriorityPolicy{{Name: "NoWeightPriority"}}},
 			expected: errors.New("priority NoWeightPriority should have a positive weight applied to it or it has overflown"),
```

Comment:
Change `ValidatePolicy` in ` pkg/scheduler/apis/config/validation/validation.go` to return a `field.Error` instead